### ISSUE TITLE
Remove search bar placeholder on focus

### ISSUE THREAD

**Issue by Android63** [NHSLeadership/nightingale#245]
Users try and select the "search" text placeholder to delete it as it occupies the space they want to fill. Suggest JS or CSS should be used to remove placeholder when search box is given focus.

**Comment by penpadstudio** [comment]
@Android63 Agreed. the placeholder search text should disappear on clicking or tapping in the box.
